api-operator/deploy/controller-configs/mgw_conf_mustache.yaml

```diff
@@ -113,6 +113,7 @@ data:
           clientId = ""
           clientSecret = ""
 
+
```

Comment:
remove additional newline

Reply:
mgw_conf_mustache.yaml file is removed in the latest master branch